bmwqemu.pm

```diff
@@ -59,6 +59,10 @@ our $istty;
 our $direct_output;
 our $standstillthreshold = scale_timeout(600);
 
+# Known locations of OVMF (UEFI) firmware: first is openSUSE, second is
+# the kraxel.org nightly packages, third is Fedora's edk2-ovmf package.
+our @ovmf_locations = ('/usr/share/qemu/ovmf-x86_64-ms.bin', '/usr/share/edk2.git/ovmf-x64/OVMF_CODE-pure-efi.fd', '/usr/share/edk2/ovmf/OVMF_CODE.fd');
```

Comment:
I prefer this split across lines, one line per file location - as in my original PR - but tidy complained about that, so we get a giganto-line.